app/assets/stylesheets/application.css

```diff
@@ -42,6 +42,22 @@ body {
     padding-bottom: 50px; /*50px for the height of the bottom navbar*/
 }
 
+a:link {
+  color: blue;
+}
+
+a:hover {
+  color: blue;
+  text-decoration: none;
+  background-color: transparent;
+}
+
+a:visited {
+  color: purple;
```

Comment:
Ich hab das damals bei meinen Projekts Links (in Project show) denen einfach eine Klasse gegeben (.link). Kannst dir das ja mal anschauen & deinen Links einfach auch diese Klasse geben